views/content/dao.md

```diff
@@ -18,316 +18,9 @@ The way this particular democracy works is that it has an **Owner** which works
-         *
-         * @param proposalNumber number of proposal
-         * @param supportsProposal either in favor or against it
-         * @param justificationText optional justification text
-         */
-        function vote(
-            uint proposalNumber,
-            bool supportsProposal,
-            string justificationText
-        )
-            onlyMembers
-            returns (uint voteID)
-        {
-            Proposal storage p = proposals[proposalNumber];         // Get the proposal
-            require(!p.voted[msg.sender]);         // If has already voted, cancel
-            p.voted[msg.sender] = true;                     // Set this voter as having voted
-            p.numberOfVotes++;                              // Increase the number of votes
-            if (supportsProposal) {                         // If they support the proposal
-                p.currentResult++;                          // Increase score
-            } else {                                        // If they don't
-                p.currentResult--;                          // Decrease the score
-            }
-
-            // Create a log of this event
-            Voted(proposalNumber,  supportsProposal, msg.sender, justificationText);
-            return p.numberOfVotes;
-        }
-
-        /**
-         * Finish vote
-         *
-         * Count the votes proposal #`proposalNumber` and execute it if approved
-         *
-         * @param proposalNumber proposal number
-         * @param transactionBytecode optional: if the transaction contained a bytecode, you need to send it
-         */
-        function executeProposal(uint proposalNumber, bytes transactionBytecode) {
-            Proposal storage p = proposals[proposalNumber];
-
-            require(now > p.votingDeadline                                            // If it is past the voting deadline
-                && !p.executed                                                         // and it has not already been executed
-                && p.proposalHash == sha3(p.recipient, p.amount, transactionBytecode)  // and the supplied code matches the proposal
-                && p.numberOfVotes >= minimumQuorum);                                  // and a minimum quorum has been reached...
-
-            // ...then execute result
-
-            if (p.currentResult > majorityMargin) {
-                // Proposal passed; execute the transaction
-
-                p.executed = true; // Avoid recursive calling
-                require(p.recipient.call.value(p.amount)(transactionBytecode));
-
-                p.proposalPassed = true;
-            } else {
-                // Proposal failed
-                p.proposalPassed = false;
-            }
-
-            // Fire Events
-            ProposalTallied(proposalNumber, p.currentResult, p.numberOfVotes, p.proposalPassed);
-        }
-    }
-
-
-
-
+```solidity
```

Comment:
This doesn't color code solidity for me. Leaving it as **\`\`\`**  or even **\`\`\`js** does the trick. Have you added a color code plugin for solidity?

Reply:
No, I haven't added any plugin for it, but I thought putting `solidity` there doesn't do any harm, BUT might inspire someone to make it happen :)
I didn't want to mark it `js` when it's not and that highlighting was a bit wonky anyway.